react-props/my-app/src/Button.tsx

```diff
@@ -0,0 +1,7 @@
+type ButtonProps = {
```

Comment:
`ButtonProps` is a fine name. FYI, I have found that just using `Props` works well and is simple.